game/src/main/java/net/demilich/metastone/game/spells/SummonSpell.java

```diff
@@ -33,9 +33,9 @@
  * made, summoning a total of {@link SpellArg#VALUE} minions.
  * <p>
  * If a {@link SpellArg#CARD_FILTER} or {@link SpellArg#CARD_SOURCE} is specified, {@link SpellArg#VALUE} minions will
- * be summoned from the generated cards, <b>without replacement</b>. Any {@link SpellArg#CARD} or {@link SpellArg#CARDS}
- * that are also specified when a filter/card source is specified will be append to the possible choices of cards to
- * summon.
+ * be summoned from the generated cards, <b>without replacement</b> (unless {@link SpellArg#RANDOM_TARGET} is also specified).
```

Comment:
this effect should just use `cards`, not even `sepcific card filter`, this change is pretty mysterious